notifications_utils/template.py

```diff
@@ -749,7 +748,7 @@ def get_html_email_body(template_content, template_values, redact_missing_person
         )
         .then(unlink_govuk_escaped)
         .then(strip_unsupported_characters)
-        .then(add_newlines_around_lang_tags)
+        # .then(add_newlines_around_lang_tags)
```

Comment:
I don't think we need this anymore, I'll remove this line completely after some more testing.